app/views/questionable/assignments/_form.html.haml

```diff
@@ -14,31 +14,31 @@
 
       - if q.accepts_multiple_answers?
         / Some inputs can take multiple answers (namely multiselects and checkboxes)
-        - if q.input_type == 'checkboxes'
+        - if q.input_type == Question::InputTypes::CHECKBOXES
           = hidden_field_tag field
           - q.options.each do |opt|
             %label.checkbox
               = check_box_tag field, opt.id, selected_option_ids.include?(opt.id)
               = opt.title
-        - elsif q.input_type == 'multiselect'
+        - elsif q.input_type == Question::InputTypes::MULTISELECT
           = select_tag field,
             options_from_collection_for_select(q.options, 'id', 'title', selected_option_ids), multiple: true
 
       - else
         / Most inputs just take a single answer
-        - if q.input_type == 'radio'
+        - if q.input_type == Question::InputTypes::RADIO
           - q.options.each do |opt|
             %label.radio
               %input{type: :radio, name: field, value: opt.id, checked: selected_option_ids.first == opt.id}
               = opt.title
         - else
           .compact-input
-            - if q.input_type == 'select'
+            - if q.input_type == Question::InputTypes::SELECT
               = select_tag "answers[#{a.id}][]",
                 options_from_collection_for_select(q.options, 'id', 'title', selected_option_ids.first), prompt: 'Choose an option'
-            - elsif q.input_type == 'string'
+            - elsif q.input_type == Question::InputTypes::STRING
               = text_field_tag field, selected_answers.first.try(:message)
-            - elsif q.input_type == 'date'
+            - elsif q.input_type == Question::InputTypes::DATE
```

Comment:
Is there a better way to use the `InputTypes` module in here too?

Reply:
Not for this one. Also I feel the extra context is useful here (a template) to track down the constant (otherwise is like angularJS stuff without controllerAs syntax).